components/rating/index.css

```diff
@@ -40,9 +81,11 @@ governing permissions and limitations under the License.
   display: inline-flex;
   flex: 0 0 auto;
 
-  inline-size: calc(var(--spectrum-rating-icon-width) * var(--spectrum-rating-icon-count));
+  inline-size: calc((var(--mod-rating-icon-width, var(--spectrum-rating-icon-width)) + var(--mod-rating-icon-spacing, var(--spectrum-rating-icon-spacing))) * var(--mod-rating-icon-count, var(--spectrum-rating-icon-count)));
```

Comment:
After you update the value of `--spectrum-rating-icon-width` above, this should look a bit more constrained. I _think_ this is the right calculation, but please check against the design file to be sure that the overall width of the component looks correct.